src/main/scala/uk/gov/hmrc/apiplatform/modules/events/applications/domain/models/ApplicationEvent.scala

```diff
@@ -287,6 +288,7 @@ object ApplicationEvent {
       case ProductionAppTermsConditionsLocationChanged(_, _, _, _, oldLocation, newLocation)                        =>
         ("T&Cs Changed", List(s"From: ${oldLocation.describe()}", s"To: ${newLocation.describe()}"))
       case ProductionLegacyAppTermsConditionsLocationChanged(_, _, _, _, oldUrl, newUrl)                            => ("T&Cs Changed", List(s"From: $oldUrl", s"To: $newUrl"))
+      case RateLimitChanged(_, _, _, _, oldRateLimit, newRateLimit)                                                 => ("Rate Limit Changed", List(s"From: $oldRateLimit", s"To: $newRateLimit"))
```

Comment:
Should we show the "pretty" names (like `Bronze`) rather than the case names (like `BRONZE`)?  Mainly a usability question, and would apply to any of these types that are enumerations. Who is the audience for this - developers or SDST?

Reply:
I think it's OK as it is. This event stuff that gets logged will be for SDST and GK users, not the TPD.